docs/resources/sys_info.md.erb

```diff
@@ -50,3 +50,15 @@ For a full list of available matchers, please visit our [matchers page](https://
 The `hostname` matcher tests the host for which standard output is returned:
 
     its('hostname') { should eq 'value' }
+
+### manufacturer
+
+The `manufacturer` matcher tests the host for which standard output is returned:
+
+    its('manufacturer') { should eq 'value' }
```

Comment:
I see you're duplicating the other examples in the same file, but it would be great if we could have semi-realistic examples of output - the word 'value' might be confusing, and we want to be clear.

Perhaps:
```
its('manufacturer') { should eq 'Acme Computers' }
```

Reply:
fixed